PatientWebApplication/HealthClinic.DAL/Model/Orders/MedicineDescription.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using HealthClinic.CL.Model.Patient;
+
+namespace HealthClinic.CL.Model.Orders
+{
+    public class MedicineDescription : Entity
+    {
+        public String Name { get; set; }
+        public String Description { get; set; }
+        public int MedicineDescriptionId { get; set; }
+
+        public MedicineDescription() : base() { }
+
+        public MedicineDescription(int id, string name, String description, int medicineID) : base(id)
```

Comment:
Change variable name from medicineID to medicineDescriptionId to match lines above